### PR TITLE
Added KOSSprint Documentation

### PR DESCRIPTION
**Pull request by harshit-jain52**
**Path(s) which will be affected by this Pull Request.**
- `/events`

### CODE REVIEW

File: events/kossprint.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# KOSSprint
+
```
Comment (harshkhandeparkar) [inline events/kossprint.md:2]:
```suggestion
KOSS internal code sprint.
```

File: events/kossprint.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+## What is KOSSprint?
+- It's an internal event held to accelerate slowed-down work, where KOSS members sit together for 2-3 hours and work on KOSS projects, slides, documentation, etc. that need immediate attention.
```
Comment (harshkhandeparkar) [inline events/kossprint.md:4]:
```suggestion
- It's an internal event held to efficiently collaborate on large projects or accelerate slowed-down work, where KOSS members sit together for 2-3 hours and work on KOSS projects, slides, documentation, etc.
```

File: events/kossprint.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+- It boosts productivity and encourage discussion among KOSS members.
```
Comment (harshkhandeparkar) [inline events/kossprint.md:5]:
```suggestion
- It boosts productivity and encourages discussion and collaboration among KOSS members.
```

File: events/kossprint.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+## When to conduct a KOSSprint?
+Sprints can be done whenever a lot of coding work (or any other work) is pending.
+eg:
+  - For planning a workshop, all can work together to make slides and poster.
+  - During KWOC, all can work together to update the frontend, backend, mailing, etc.
```
Comment (harshkhandeparkar) [inline events/kossprint.md:11]:
```suggestion
  - During KWoC, all can work together to update the frontend, backend, mailing, etc.
```

File: events/kossprint.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+
+## Where can KOSSprint be conducted?
+The place should have good internet connection, enough plug points, no time-related restrictions (as sprints are usually held around midnight).
```
Comment (harshkhandeparkar) [inline events/kossprint.md:14]:
```suggestion
The place should have a good internet connection, enough plug points, and no time restrictions (as sprints are usually held around midnight).
```

File: events/kossprint.md [hunk 6]
```diff
@@ -0,0 +1,18 @@
+# KOSSprint
+
+## What is KOSSprint?
+- It's an internal event held to accelerate slowed-down work, where KOSS members sit together for 2-3 hours and work on KOSS projects, slides, documentation, etc. that need immediate attention.
+- It boosts productivity and encourage discussion among KOSS members.
+
+## When to conduct a KOSSprint?
```
Comment (harshkhandeparkar) [inline events/kossprint.md:7]: Also add suggested dates (weekends) and times (10pm - 1am) under this section.

**Review by harshkhandeparkar — CHANGES_REQUESTED** [review]

File: events/kossprint.md [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+- Suggested days:
+  - Weekends
+- Suggested times:
+  - 10pm - 1am  
+
+## Where can KOSSprint be conducted?
+- The place should have a good internet connection, enough plug points, and no time restrictions (as sprints are usually held around midnight).
+- Some suggested venues:
+  - Maggu Room
+  -
```
Comment (harshkhandeparkar) [inline events/kossprint.md:22]: Do you know any other venue? If not, this can be removed for now and updated in the future.

Comment (harshit-jain52) [reply events/kossprint.md:22]: library? :skull:

Comment (harshit-jain52) [reply events/kossprint.md:22]: corridors of main building first floor

Comment (harshkhandeparkar) [reply events/kossprint.md:22]: > library? 💀

Ig we can add this.

Comment (harshkhandeparkar) [reply events/kossprint.md:22]: > corridors of main building first floor

Are sockets available?

Comment (harshit-jain52) [reply events/kossprint.md:22]: hm, no sockets in corridors

okay, library added

**Review by harshkhandeparkar — APPROVED** [review]

**Comment by harshit-jain52** [comment]
Merge? :partying_face: :

**Comment by harshkhandeparkar** [comment]
Thanks @harshit-jain52! :rocket:

**Pull request merged**